src/components/Settings/SlippageTolerance.vue

```diff
@@ -62,11 +64,12 @@ import { Components } from '@/consts'
   }
 })
 export default class SlippageTolerance extends Mixins(TranslationMixin, NumberFormatterMixin) {
-  readonly SlippageToleranceTabs = [
+  readonly delimiters = FPNumber.DELIMITERS_CONFIG
+  SlippageToleranceTabs: any = [
     0.1,
```

Comment:
maybe array with strings? and please make it `readonly` and `Array<string>` type